docs/software/sciapps/namd.md

```diff
@@ -205,52 +207,137 @@ The multi-node build provides the following views:
 !!! note "GPU-resident mode"
     The multi-node build based on [Charm++]'s MPI backend can't take advantage of the new GPU-resident mode. Unless you require the multi-node
     build or you can prove it is faster for your use case, we recommend using the single-node build with the GPU-resident mode.
+
+
+### Running NAMD on Eiger
+
+The following sbatch script shows how to run NAMD on Eiger:
+
+```bash
+#!/bin/bash -l
+#SBATCH --job-name=namd-test
+#SBATCH --time=00:30:00
+#SBATCH --nodes=4
+#SBATCH --ntasks-per-core=1
+#SBATCH --ntasks-per-node=128
+#SBATCH --account=<ACCOUNT> (1)
+#SBATCH --hint=nomultithread
+#SBATCH --hint=exclusive
+#SBATCH --constraint=mc
+#SBATCH --uenv=namd/3.0:v1 (2)
+#SBATCH --view=namd (3)
+
+export OMP_NUM_THREADS=$SLURM_CPUS_PER_TASK
+export OMP_PROC_BIND=spread
+export OMP_PLACES=threads
+
+srun --cpu-bind=cores namd3 +setcpuaffinity ++ppn 4 <NAMD_CONFIG_FILE> # (4)!
+```
+
+1. Change `<ACCOUNT>` to your project account
+2. Load the NAMD UENV (UENV name or path to the UENV). Change `<NAMD_UENV>` to the name (or path) of the actual NAMD UENV you want to use
+3. Load the `namd` view
+4. Make sure you set `++ppn`, and other NAMD options optimally for your calculation.
```

Comment:
Out of ignorance: is the double `++` intentional?
```suggestion
4. Make sure you set `++ppn`, and other NAMD options optimally for your calculation.
```

Reply:
Yes, this is intended. I'm still testing a bit to see what is the optimal configuration for Eiger (documentation seems a bit scarce for multicore).